_articles/legal.md

```diff
@@ -90,7 +90,7 @@ For example, as your project grows it adds dependencies or users, or your compan
 
 Alternatively, you can have contributors agree in advance (via an additional contributor agreement -- see below) to certain license changes under certain conditions, beyond those allowed by your existing open source license. This shifts the complexity of changing licenses a bit. You'll need more help from your lawyers up front, and you'll still want to clearly communicate with your project's stakeholders when executing a license change.
 
-## Does my project need an additional contributor agreement?
+## Does my project need an additional contributor [license] agreement (CLA)?
```

Comment:
The titles are already really long on most of these guides. Could we keep this title the same, and call this out in the paragraph text?